lib/src/widgets/chat/video/ds_video_message_bubble.widget.dart

```diff
@@ -120,57 +131,95 @@ class _DSVideoMessageBubbleState extends State<DSVideoMessageBubble>
+                                          maximumProgress:
+                                              _controller.maximumProgress,
+                                          foregroundColor: foregroundColor,
+                                        )
+                                      : _controller.thumbnail.isEmpty
+                                          ? Center(
+                                              child: SizedBox(
+                                                height: 40,
+                                                child: DSButton(
+                                                  leadingIcon: const Icon(
+                                                    DSIcons.download_outline,
+                                                    size: 20,
+                                                  ),
+                                                  backgroundColor:
+                                                      buttonBackgroundColor,
+                                                  foregroundColor:
+                                                      buttonForegroundColor,
+                                                  borderColor:
+                                                      buttonBorderColor,
+                                                  label: _controller.size(),
+                                                  onPressed:
+                                                      _controller.downloadVideo,
+                                                ),
+                                              ),
+                                            )
+                                          : DSVideoBody(
+                                              align: widget.align,
+                                              appBarPhotoUri:
+                                                  widget.appBarPhotoUri,
+                                              appBarText: widget.appBarText,
+                                              url: widget.url,
+                                              shouldAuthenticate:
+                                                  widget.shouldAuthenticate,
+                                              thumbnail: Center(
+                                                child: Image.file(
+                                                  File(
+                                                    _controller.thumbnail.value,
+                                                  ),
+                                                  width: 240,
+                                                  height: 240,
+                                                  fit: BoxFit.cover,
+                                                ),
+                                              ),
+                                            ),
+                        )
+                      : const SizedBox.shrink(),
```

Comment:
In this case, I think you should show the same Icon displayed when ```_controller.hasError.value``` is ```true```. Makes sense?